apps/web/pages/video/[uid].tsx

```diff
@@ -60,11 +79,64 @@ export default function JoinCall(props: JoinCallPageProps) {
       url: meetingUrl,
       ...(typeof meetingPassword === "string" && { token: meetingPassword }),
     });
+    //TODO - add handlers for `transcription-started` and `transcription-ended` @link https://docs.daily.co/reference/rn-daily-js/events/transcription-events
+    /** handling custom-button-click @link  https://docs.daily.co/reference/daily-js/events/meeting-events#custom-button-click */
+    const onCustomButtonClick = async (event?: DailyEventObjectCustomButtonClick | undefined) => {
+      console.log(`calAiCredential ${JSON.stringify(props.calAiCredential)}`);
+
+      if (!props.calAiCredential || !props.calAiCredential.key || !props.calAiCredential.key["apiKey"]) {
```

Comment:
- This is causing a type error
- Do we want credentials keys on the frontend? This makes them visible to anyone inspecting the HTML code.